x-pack/plugins/security_solution/public/exceptions/components/import_exceptions_list_flyout/index.tsx

```diff
@@ -128,6 +132,13 @@ export const ImportExceptionListFlyout = React.memo(
             importExceptionListState?.result?.errors.forEach((err) => {
               if (err.error.message.includes('already exists')) {
                 setAlreadyExistingItem(true);
+                if (
+                  err.error.message.includes(
+                    `Found that list_id: "${ENDPOINT_LIST_ID}" already exists`
```

Comment:
I think it's not an ideal way, of parsing exception messages like that, because the message can change, and this code will not work after.

Do we have any type or other identifier from API?

Reply:
Unfortunately, there is no type. We only create a message with the list ID which already exists and pass it as an error message here https://github.com/elastic/kibana/blob/main/x-pack/plugins/lists/server/services/exception_lists/utils/import/sort_exception_lists_to_create_update.ts#L135